Sources/JXKit/JXContext.swift

```diff
@@ -4,24 +4,63 @@ import JavaScriptCore
 #else
 import CJSCore
 #endif
-#if canImport(FoundationNetworking)
-import FoundationNetworking
+#if canImport(Foundation)
+import Foundation
 #endif
 
 /// A JavaScript execution context. This is a cross-platform analogue to the Objective-C `JavaScriptCore.JSContext`.
 ///
 /// This wraps a `JSGlobalContextRef`, and is the equivalent of `JavaScriptCore.JSContext`
 open class JXContext {
+    /// Context configuration.
+    public struct Configuration {
+        /// Whether scripts evaluated by this context should be assessed in `strict` mode. Defaults to `true`.
+        public var strict: Bool
+        
+        /// Whether hot reloading of JavaScript script resources is enabled.
+        public var isHotReloadEnabled: Bool {
```

Comment:
We could just call this "Reload" rather than "HotReload"

Reply:
Yeah, and maybe we could expression modes with an OptionSet like `context.options.loadOptions = [.hotReload, .versionFreezing]`. But for the time being, a single `isHotReloadEnabled` property is fine (although I might prefer a clearer name than "hot", something like "dynamic", as in `isDynamicReloadEnable`).